cosinnus_notifications/views.py

```diff
@@ -233,11 +233,13 @@ def get_context_data(self, **kwargs):
                 'multi_preference_setting': UserMultiNotificationPreference.get_setting_for_user(self.request.user, multi_notification_id),
             })
         
-        # get conferences user is pending for or is a member of
+        # get conferences with application method `Request application` the user is pending for or is a member of
         user = self.request.user
         user_conferences = CosinnusConference.objects.get_for_user_pks(user)
         pending_application_qs = CosinnusConferenceApplication.objects.filter(user=user).pending().filter(may_be_contacted=True)
-        subscribed_conferences = list(set(CosinnusConference.objects.filter(Q(conference_applications__in=pending_application_qs) | Q(id__in=user_conferences))))
+        subscribed_conferences = list(set(CosinnusConference.objects.filter(
```

Comment:
The conferences listed should only be the ones where the user has `may_be_contacted` checked. Your filter only checks that boolean field on *pending* applications, but not on applications of conferences where the user has been accepted as member already. You need to also add that filter as well, else the list would show all accepted conferences with membership_mode_application for the user.